cmd/up.go

```diff
@@ -123,3 +141,21 @@ func isGKE(disco discovery.DiscoveryInterface) (bool, error) {
 
 	return false, nil
 }
+
+func generateMongoDBSecret(cli kubernetes.Interface, objs []*unstructured.Unstructured) error {
+	var mongoNS string
+	for _, obj := range objs {
```

Comment:
This will produce the right answer, but we *know* the answer is always going to be "kubeapps", right?

(as in: we "know" it as strongly as we know that we need to generate a mongodb secret with these exact fields, etc.  Reactive/robust code is good, but the whole purpose of this function is to make some assumptions about what we're deploying.  Why not just also assume the namespace?)